CMSgov/onemac
# OY2-13092: build package filtering UI

Story: https://qmacbis.atlassian.net/browse/OY2-13092
Endpoint: https://d1sz7p5plj5p8v.cloudfront.net/

### Details

Implement "filter" UI on Submission Dashboard

### Changes

- Add a "Filter" button to search bar component that toggles open a sidebar menu
- Implement said sidebar menu with open/close functionality, filter accordions with checkboxes, and a "Reset" button
- Adjust application styles to support this new layout

### Implementation Notes

- Implemented without "Apply" button as this was easier. HCD approved.
- Put menu on right side rather than left. HCD approved.
- This initial implementation has not done much to provision our future filters (territories and dates). We will cross that proverbial bridge when we get there.

### Test Plan

1. Log in as `statesubmitteractive` and navigate to the Packages link. If there are no submissions in the table, create some to work with - preferably with multiple package types.
2. Once ready, click the new "Filter" button on the left above the table and take a look at the new filter panel. Verify that the cosmetic and interactive experience is in line with the Figma (modulo the exceptions noted above).
3. Verify that unchecking options removes the relevant packages from view in the table.
